src/vm/i386/cgenx86.cpp

```diff
@@ -1730,6 +1730,30 @@ DWORD getextcpuid(DWORD arg1, DWORD arg2, unsigned char result[16])
     }
 }
 
+extern "C"
+__declspec(naked)
```

Comment:
I'm not well-versed in how __asm works. I copy-and-pasted the code from the amd64 asm file, and making it naked allowed that to work. If I made it non-naked, what would I do? Remove the "ret" but leave everything else? I had a problem before where there were contracts getting in the way, too.